### PR TITLE
added task & tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/doubleTrouble.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test('doubleTrouble([3, 7, 3, 9], 8)', () => {
+  expect(doubleTrouble([3, 7, 3, 9], 8))
+    .toEqual([3, 7, 3, 9]);
+});
+
+test(
+  `doubleTrouble([6, 4, 9, 6, 4, 5, 2, 2, 9, 6, 4, 1, 9, 8, 7, 7, 5, 2, 5, 6, 
+  3, 2, 9, 5, 9, 7, 4, 3, 2], 9)`, () => {
+    expect(doubleTrouble([6, 4, 9, 6, 4, 5, 2, 2, 9, 6, 4, 1, 9, 8, 7, 7,
```
Comment (roman-mate) [inline src/doubleTrouble.test.js]: If arguments don't fit into one line, we should do first line break before the first argument (after "doubleTrouble(" )

File: src/doubleTrouble.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * Function takes an array of integers (array), and a target (target), you must
+ * find out if any two consecutive numbers in the array sum to target. If so,
+ * remove the second number.
+ *
+ * Example:
+ *
+ * const array = [1, 2, 3, 4, 5]
+ * const target = 3
+ *
+ * 1+2 = target, so remove 2. No other pairs = target, so rest of array remains:
```
Comment (mgrinko) [inline src/doubleTrouble.js]: @ivadimko what is implemented in tests?
1) there is no information about the position to start from
If we start checking from the end it with be quite another result [1, 2, 1]
2) should we recheck after a removing a number?

Comment (ivadimko) [reply src/doubleTrouble.js]: @mgrinko line below
`Work through the array from left to right.`

another part of the description:
`if any two consecutive numbers in the array sum to target`

So we start at 0, move through the array and check `next number` + `current number`. If `next + current = target` we remove `next`.

Comment (ivadimko) [reply src/doubleTrouble.js]: https://www.codewars.com/kata/double-trouble

Comment (ivadimko) [reply src/doubleTrouble.js]: done in another PR without approving because it's minor changes

**Pull request merged**